.changeset/blue-frogs-rush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@talend/design-system': minor
```

Comment:
This looks to be a breaking change for design-system.
Also, there is changes also in form packages but without a changeset

Reply:
Not really a change there, correct? I only updated the way the prop is passed.